core/src/main/scala/cats/syntax/TrySyntax.scala

```diff
@@ -0,0 +1,29 @@
+package cats.syntax
+
+import cats.Applicative
+import cats.data.EitherT
+
+import scala.util.{ Failure, Success, Try }
+
+trait TrySyntax {
+  implicit final def catsSyntaxTry[A](t: Try[A]): TryOps[A] = new TryOps[A](t)
+}
+
+final class TryOps[A](val t: Try[A]) extends AnyVal {
+
+  /**
+    * Converts a `Try[A]` to a `Either[Throwable, A]`.
+    *
+    * @see [[cats.syntax.EitherObjectOps#fromTry]]
+    */
+  def toEither: Either[Throwable, A] = t match {
```

Comment:
Just mentioning that this syntax is already provided by [mouse](https://github.com/typelevel/mouse/blob/master/shared/src/main/scala/mouse/try.scala#L17).
There is no real clear line though for what belongs in Cats itself or what in mouse.

Reply:
good point. I'm happy to close this without merging. I can add `toEitherT` to mouse instead.